delivery_carrier_label_ups/models/stock_picking.py

```diff
@@ -167,9 +164,68 @@ def _ups_shipping_data(self):
                     ],
                     **shipping_data,
                 ),
+                LabelSpecification=self._ups_label_specification(account),
             )
         )
 
+    def _ups_return_service(self):
+        """Return a dict describing the UPS Return Label code"""
+        self.ensure_one()
+        recipient = self.picking_type_id.warehouse_id.partner_id
+        email = recipient.email
+        if recipient.country_id.code != 'US':
+            subject_code = '01'  # First shipment reference
+        else:
+            subject_code = '03'  # First package reference
+
+        return dict(
+            ReturnService=dict(
+                # UPS Electronic Return Label (ERL)
+                Code='8',
+            ),
+            ShipmentServiceOptions=dict(
+                LabelDelivery=dict(
+                    EMailMessage=dict(
+                        EMailAddress=email,
+                        FromEMailAddress=email,
+                        SubjectCode=subject_code,
+                    )
+                ),
+                Notification=[
+                    dict(
+                        NotificationCode='2',
+                        EMail=dict(
+                            EMailAddress=email,
+                            UndeliverableEMailAddress=email,
```

Comment:
Sorry if I'm being redundant, but the undeliverable email address should also be set to the sender email if you ask me.